.github/workflows/precommit.yml

```diff
@@ -66,7 +66,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Run smoke tests
-        run: docker run --rm toolsolutions-pytorch:latest ./test.sh
+        run: docker run --rm toolsolutions-pytorch:latest ./test-examples.sh
+
+      - name: Run unit tests
+        run: docker run --rm toolsolutions-pytorch:latest bash ./run_unit_tests.sh
```

Comment:
This one calls the script via bash but the other calls it directly, it would be great to make this consistent, preferrably by making `run_unit_tests.sh` executable like `test-examples.sh`